recipes/sphinx-autobuild/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "sphinx-autobuild" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "2f9262d7a35f80a18c3bcb03b2bf5a83f0a5e88b75ad922b3b1cee512c7e5cd2" %}
+
+package:
+  name: sphinx-autobuild
+  version: "0.6.0"
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - sphinx-autobuild=sphinx_autobuild:main
+
+requirements:
+  build:
+    - python
```

Comment:
The `run` section is fine as is. What I was saying was that the `build` sections should just look like:

```
build:
   - python
   - setuptools
```

since none of the other packages are actually needed to build the recipe.